### ISSUE TITLE
Change Wazuh AMI OS to AL2023

### ISSUE THREAD

**Issue by teddytpc1** [wazuh/wazuh-packages#2986]
## Description
We need to change the Wazuh AMI OS to AL2023

## Tasks
- [ ] ~~Generate an AL2023 AMI to be used for the Wazuh AMI~~
- [ ] Test the Wazuh AMI build using the new AL2023 AMI with the pipeline
- [ ] Validate the new Wazuh AMI works.

NOTE: the tests can be made using the 4.8.0 packages.

**Comment by davidcr01** [comment]
# Update Report

## Next steps

- [ ] Update AL2023 AMI specified in the `Packages_builder_AMI.groovy` pipeline.
- [ ] Test AMI generated by the pipeline.

**Comment by davidcr01** [comment]
This issue goes to Blocked due to: https://github.com/wazuh/wazuh-packages/issues/2985#issuecomment-2162443265 until we decide the workaround.

**Comment by davidcr01** [comment]
**On Hold due to allocator issues.**

**Comment by davidcr01** [comment]
Blocked by: https://github.com/wazuh/wazuh-virtual-machines/issues/122. OVA can't not be migrated to AL2023 and the OVA and AMI should be built in the same OS.

**Comment by teddytpc1** [comment]
AWS reply:
```
The service team have informed me that currently, they do not have an update when the OS will support this feature using the EC2 Import / Export option. 

Any new feature announcements will be communicated via email or can be found here: https://aws.amazon.com/new/?whats-new-content-all.sort-by=item.additionalFields.postDateTime&whats-new-content-all.sort-order=desc&awsf.whats-new-categories=*all 
```
